src/main/java/frc/robot/subsystems/Pneumatics.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.CompressorConfigType;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.PneumaticsConstants;
+
+/**
+ * A subsystem that controls pneumatics on a robot.
+ */
+public class Pneumatics extends SubsystemBase {
+  private final Compressor compressor;
+
+  /**
+   * Creates a Pneumatics subsystem with the default module.
+   */
+  public Pneumatics() {
+    compressor = new Compressor(PneumaticsModuleType.REVPH);
+    this.setAnalog();
+  }
+
+  /**
+   * Creates a Pneumatics subsystem with a specified compressor module.
+   * 
+   * @param module The module of the compressor
+   */
+  public Pneumatics(int module) {
```

Comment:
Remove this. If there are multiple PCM's on the robot, they will all be on the same subsystem.